src/cephadm/cephadm

```diff
@@ -6710,23 +6709,30 @@ class HostFacts():
         # type: () -> Dict[str, str]
         """Determine the security features enabled in the kernel - SELinux, AppArmor"""
         def _fetch_selinux() -> Dict[str, str]:
-            """Read the selinux config file to determine state"""
+            """Get the selinux status"""
             security = {}
-            for selinux_path in HostFacts._selinux_path_list:
-                if os.path.exists(selinux_path):
-                    selinux_config = read_file([selinux_path]).splitlines()
-                    security['type'] = 'SELinux'
-                    for line in selinux_config:
-                        if line.strip().startswith('#'):
-                            continue
-                        k, v = line.split('=')
-                        security[k] = v
-                    if security['SELINUX'].lower() == 'disabled':
-                        security['description'] = 'SELinux: Disabled'
-                    else:
-                        security['description'] = 'SELinux: Enabled({}, {})'.format(security['SELINUX'], security['SELINUXTYPE'])
-                    return security
-            return {}
+            try:
+                out, err, code = call(self.ctx, ['sestatus'],
+                                        verbosity=CallVerbosity.DEBUG)
```

Comment:
> but that doesn't make sense given the code and didn't show up when I ran tox on these changes locally so hopefully the error will go away if we rerun it.

Actually, I think it might have been complaining about indent on this line

```suggestion
                out, err, code = call(self.ctx, ['sestatus'],
                                      verbosity=CallVerbosity.DEBUG)
```

Reply:
Done!